### DISCUSSION TITLE
How to use KiBOT in Github Actions with KiCAD V8 ?

### DISCUSSION THREAD

**Discussion by Strooom** [INTI-CMNB/KiBot#666] [Q&A] [answered]
I am experimenting with adding CI/CD to my KiCAD designs. 
So I added this basic workflow file https://github.com/Strooom/LoRa-V3-Mini-PCB/blob/main/.github/workflows/drcerc.yml
KiBOT throws an error:

`ERROR:In preflight erc: The erc preflight needs KiCad 8 or newer, use run_erc instead (kibot.gs - gs.py:845)`

and at the top of the logging It looks like it installed KiCAD 5.1.9 instead of V8.x (I would like to use the latest..)
```
DEBUG:KiBot 1.8.0 verbose level: 1 started on 2024-09-23 12:03:07.093898 (kibot - __main__.py:462)
DEBUG:Filtering warnings we always get on CI/CD (kibot - __main__.py:401)
DEBUG:Detected KiCad v5.1.9 (5.1.9+dfsg1-1+deb11u1 [50](https://github.com/Strooom/LoRa-V3-Mini-PCB/actions/runs/10993818331/job/30521112833#step:4:51)010090) (kibot - __main__.py:304)
```

My designs are V8.x

How do I make this setup use KiCAD V8.x ??

The repository is public at https://github.com/Strooom/LoRa-V3-Mini-PCB

**Comment by set-soft** [comment, marked as answer]
The available tags for the GitHub Action are listed [here](https://kibot.readthedocs.io/en/latest/usage_with_ci_cd.html#github-actions-tags)

Which is part of the [Usage of GitHub Actions](https://kibot.readthedocs.io/en/latest/usage_with_ci_cd.html#index-3) section (the link might move with time, but the page is the same)

So you should replace `v2` by `v2_k8`.

**Reply by Strooom** [reply]
Thank you - for some reason I missed those doc pages.
